core/src/main/scala/scala/pickling/spi/PicklerRegistry.scala

```diff
@@ -56,23 +56,22 @@ trait PicklerRegistry {
     *             In those situations, the pickler should be able to handle arbitrary (existential) type parameters.
     * @param p  The pickler to register.
     */
-  def registerPickler[T](key: String, p: Pickler[T]): Unit
-  def registerPickler[T](p: Pickler[T]): Unit = registerPickler(p.tag.key, p)
+  def registerPickler(key: String, p: Pickler[_]): Unit
+  def registerPickler(p: Pickler[_]): Unit = registerPickler(p.tag.key, p)
```

Comment:
Yes.   Although, when you say "powerful abstraction", I'm not sure what the difference between:

`def foo(x: A forSome { type A }): Unit` or `def foo[A](x: A): Unit` is....

I think perhaps we disagree on which one is less powerful, practically, they both accept all types :)